skydriver/database/__init__.py

```diff
@@ -1,3 +1,23 @@
 """Init."""
 
-from . import interface, schema  # noqa: F401
+
+from urllib.parse import quote_plus
+
+from motor.motor_asyncio import AsyncIOMotorClient  # type: ignore
+
+from ..config import ENV
+from . import interface
+
+
+async def create_mongodb_client() -> AsyncIOMotorClient:
+    """Construct the MongoDB client."""
+    auth_user = quote_plus(ENV.MONGODB_AUTH_USER)
+    auth_pass = quote_plus(ENV.MONGODB_AUTH_PASS)
+
+    url = f"mongodb://{ENV.MONGODB_HOST}:{ENV.MONGODB_PORT}"
+    if auth_user and auth_pass:
+        url = f"mongodb://{auth_user}:{auth_pass}@{ENV.MONGODB_HOST}:{ENV.MONGODB_PORT}"
+
+    mongo_client = AsyncIOMotorClient(url)
+    await interface.ensure_indexes(mongo_client)
```

Comment:
Just something to be aware of: this will block while the indexes are created.  Not a problem for an empty DB, but when it has stuff in it then it can be an issue.  I hit this a few times with the FC and iceprod, wondering why the server was running but not accessible yet.  My solution was to do index creation in a background task.

Reply:
good point, [fefe5dd](https://github.com/WIPACrepo/SkyDriver/pull/36/commits/fefe5dd63bcfe5728b70736319807aa763bebc10)